OutOfSchool/OutOfSchool.WebApi/Services/GeocodingService.cs

```diff
@@ -241,5 +241,11 @@ private static Point CalculateSouthEast(double lat, double lon, double deltaMete
         return new Point(lat - (deltaMeters * Coef), lon + (deltaMeters * Coef / Math.Cos(GeoMathHelper.Deg2Rad(lat))));
     }
 
-    private sealed record Point(double lat, double lon);
+    private sealed record Point(double lat, double lon)
+    {
+        public override string ToString()
+        {
+            return $"{lon.ToStringWithDotSeparator()} {lat.ToStringWithDotSeparator()}";
```

Comment:
Consider using the CultureInfo or NumberFormatInfo instead of creating an extension method.